core/src/main/java/org/apache/brooklyn/entity/group/DynamicClusterImpl.java

```diff
@@ -485,6 +491,17 @@ protected void doStart() {
         }
     }
 
+    public void waitForServiceUp() {
```

Comment:
Good point - will change to protected (I copied it from `SoftwareProcessImpl`, where these are public).